src/GithubFilesBundle/Controller/DefaultController.php

```diff
@@ -19,18 +19,22 @@ class DefaultController extends Controller
     /**
      * @Route("/user/github/repos")
      */
-    public function reposAction()
+    public function getRepos(Request $request)
```

Comment:
route actions should end with "Action", so this should probably be renamed to getReposAction